CMakeLists.txt

```diff
@@ -353,13 +390,28 @@ protobuf_generate_cpp(PROTO_SRCS PROTO_HDRS ${PROTO_FILES})
 message("pika PROTO_SRCS = ${PROTO_SRCS}")
 message("pika PROTO_HDRS = ${PROTO_HDRS}")
 
-add_executable(${PROJECT_NAME}
+add_library(${PROJECT_NAME}_static
   ${DIR_SRCS}
   ${PROTO_SRCS}
   ${PROTO_HDRS}
   ${PIKA_BUILD_VERSION_CC})
 
+target_include_directories(${PROJECT_NAME}_static
+  PUBLIC ${CMAKE_CURRENT_BINARY_DIR}
+  PUBLIC ${PROJECT_SOURCE_DIR}
+  PUBLIC src
+  PUBLIC src/blackwidow/include
+  ${INSTALL_INCLUDEDIR}
+)
+
+add_executable(${PROJECT_NAME} src/pika.cc)
+
+target_link_directories(${PROJECT_NAME}
+  PUBLIC ${INSTALL_LIBDIR_64}
+  PUBLIC ${INSTALL_LIBDIR})
+
 add_dependencies(${PROJECT_NAME}
```

Comment:
add_dependencies() have to be moved to ${PROJECT_NAME}_static which using `glog` etc.